### ISSUE TITLE
Add Delphi-Pascal with test data

### ISSUE THREAD

**Issue by kotp** [exercism/seeds#15]
The issue exercism/exercism.io#3453 indicates that there should be a change in this repository.

**Comment by kytrinyx** [comment]
Yes, it's been a long time since I did any work on the seeds, I can't remember how well it's documented. I guess I'll find out.
